backend/services/implementations/__tests__/schoolService.test.ts

```diff
@@ -123,4 +125,25 @@ describe("mongo schoolService", (): void => {
       await schoolService.createSchool(testSchoolInvalidTeacher);
     }).rejects.toThrowError("One or more of the teacher IDs was not found");
   });
+
+  it("update school for valid schools", async () => {
+    // add test school
+    const savedId = await SchoolModel.create(testSchool);
```

Comment:
```suggestion
    const school = await SchoolModel.create(testSchool);
```
nit: rename for clarity